tmt/cli.py

```diff
@@ -1866,12 +1866,16 @@ def status(
         raise tmt.utils.GeneralError(
             "Options --abandoned, --active and --finished cannot be "
             "used together.")
-    if not Path(workdir_root).exists():
+
+    workdir_root = Path(_workdir_root) if _workdir_root is not None else None
+    if workdir_root and not workdir_root.exists():
         raise tmt.utils.GeneralError(f"Path '{workdir_root}' doesn't exist.")
 
     status_obj = tmt.Status(
-        logger=context.obj.logger.clone().apply_verbosity_options(**kwargs),
-        cli_invocation=CliInvocation.from_context(context))
+        logger=context.obj.logger.clone().
+        apply_verbosity_options(**kwargs),
```

Comment:
>> Is the new line really needed here?
> I guess so, we need pass workdir_root to Status class,or, let's say,Common
class, so that we can get  self.workdir_root there:)

@skycastlelily the question is about the new-line in `logger` argument. Would this be better?

```
        logger=context.obj.logger.clone().apply_verbosity_options(**kwargs),
```

AFAICT the first two parameters to `tmt.Status` did not really change, right? `logger=context.obj.logger.clone().apply_verbosity_options(**kwargs)` and `cli_invocation=CliInvocation.from_context(context)` lines should not change by this patch, except the trailing `)` would be replaced with `,`. This is how it would look:

```python
    status_obj = tmt.Status(
        logger=context.obj.logger.clone().apply_verbosity_options(**kwargs),
        cli_invocation=CliInvocation.from_context(context),
        workdir_root=effective_workdir_root(workdir_root))
```

Reply:
Ah,got it now ,thanks for mentor's explain^^